<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkingField</class>
 <widget class="QWidget" name="WorkingField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>623</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>배치</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="3,0,3">
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>미지정 교사</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,0,1,0,0,1">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>관내 전보:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_inside">
       <property name="text">
        <string>0</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>명</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>관외 전보:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_outside">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>명</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="unspecified_tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>관내</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_internal">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>11</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>전보유형</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>소속</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>성명</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>성별</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>급지</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>전보년수</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>전보점수</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>1지망</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2지망</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>3지망</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>비고</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>관외</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_external">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>전보유형</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>지역</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>학교명</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>직위</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>성명</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>생년월일</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>성별</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>교육총경력</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>심화전공</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>1희망지구</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>2희망지구</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>3희망지구</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>휴직종류</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>휴직시작일</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>휴직종료일</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>친인척학교명</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>관계</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>성명</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>주소</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>휴대폰</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>이메일</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>교통수단</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>비고</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>id</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>추가&gt;&gt;</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_del">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;&lt;삭제</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="schoolListWidget">
       <property name="styleSheet">
        <string notr="true">QListWidget{
	border: 0px;
	background-color: rgb(246, 246, 246);
	selection-color: rgb(0, 0, 0);
     selection-background-color: rgb(220, 220, 220);
	font: 18pt &quot;Nanum Gothic&quot;;
}
</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>전입자</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>전출자</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget_gone">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_add</sender>
   <signal>clicked()</signal>
   <receiver>WorkingField</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>573</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_del</sender>
   <signal>clicked()</signal>
   <receiver>WorkingField</receiver>
   <slot>delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>schoolListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>WorkingField</receiver>
   <slot>tabChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>636</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WorkingField</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>936</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>schoolListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>schoolListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget_gone</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>685</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>delete()</slot>
  <slot>tabChanged(int)</slot>
  <slot>save()</slot>
 </slots>
</ui>
